src/test/java/seedu/ccacommander/testutil/TypicalHours.java

```diff
@@ -0,0 +1,14 @@
+package seedu.ccacommander.testutil;
+
+import seedu.ccacommander.model.enrolment.Hours;
+
+/**
+ * A utility class containing a list of {@code Hours} objects to be used in tests.
+ */
+public class TypicalHours {
+    public static final Hours HOURS_ZERO = new Hours("0");
+    public static final Hours HOURS_ONE = new Hours("1");
+    public static final Hours HOURS_TEN = new Hours("10");
+    public static final Hours HOURS_MAX = new Hours("2147483647");
```

Comment:
since only Hours_zero is used for the tests in this PR, maybe we can keep in mind to use the other three in future test cases or remove the other ones. I think we sometimes have a habit of introducing more variables than needed in tests so it might clutter the test files.

Reply:
Agreed! But we still have time to clear up the test files and streamline our test cases in v1.4, so no rush for now.